src/Symfony/Component/Config/Util/XmlUtils.php

```diff
@@ -152,7 +152,7 @@ public static function loadFile($file, $schemaOrCallable = null)
      * @param \DOMElement $element     A \DOMElement instance
      * @param bool        $checkPrefix Check prefix in an element or an attribute name
      *
-     * @return array A PHP array
+     * @return mixed
```

Comment:
This method can return virtually any non-object type and we have tests covering this. That is a bit surprising, given that the method name ends with `ToArray`.